packages/datadog-plugin-aws-sdk/src/helpers.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const DEFAULT = 'default'
+const Services = {
+  cloudwatchlogs: require('./services/cloudwatchlogs'),
+  dynamodb: require('./services/dynamodb'),
+  kinesis: require('./services/kinesis'),
+  s3: require('./services/s3'),
+  redshift: require('./services/redshift'),
+  sns: require('./services/sns'),
+  sqs: require('./services/sqs'),
+  [DEFAULT]: require('./services/base')
+}
+
+function getService (serviceName) {
+  if (Services[serviceName]) {
+    return new Services[serviceName]()
```

Comment:
If the variable was named with an uppercase first later because of ESLint, just setting another variable would fix it like `Service = services[name]`.